auth/src/test/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivityTest.java

```diff
@@ -146,18 +146,16 @@ public void testFacebookLoginFlow() {
         assertNotNull(facebookButton);
         facebookButton.performClick();
 
-        ShadowActivity.IntentForResult nextIntent =
-                Shadows.shadowOf(authMethodPickerActivity).getNextStartedActivityForResult();
-        verifySaveCredentialIntent(nextIntent, FacebookAuthProvider.PROVIDER_ID);
+        Intent smartLockIntent = SmartLock.getInstance(authMethodPickerActivity).getIntentForTest();
```

Comment:
These tests verify the intent when the intent is only created for the purposes of the test. It would be nice to update the tests to reflect the changes in SmartLock, ideally checking that the SmartLock API is called with the correct credential.